Angular/src/app/landing-page/landing-page.component.spec.ts

```diff
@@ -1,14 +1,26 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { LandingPageComponent } from './landing-page.component';
+import { NavbarComponent } from '../navbar/navbar.component';
+import { RouterTestingModule } from '@angular/router/testing';
+import { AdminAuthGuardComponent } from '../admin-auth-guard/admin-auth-guard.component';
+import { AuthGuardComponent } from '../auth-guard/auth-guard.component';
+import { CookieService } from 'ngx-cookie-service';
 
 describe('LandingPageComponent', () => {
   let component: LandingPageComponent;
   let fixture: ComponentFixture<LandingPageComponent>;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ LandingPageComponent ]
+      declarations: [ LandingPageComponent,
+                      NavbarComponent
+                    ],
+      imports: [ RouterTestingModule ],
+      providers: [ AdminAuthGuardComponent,
+                   AuthGuardComponent,
+                   CookieService
+                 ]
```

Comment:
I prefer the formatting like this: https://github.com/sapphireduffy/KainosCareerLattice/pull/39/files#diff-5ecc1ba348aae4edd8273433f83d029aR17